components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -171,7 +171,7 @@ - (void)setRippleHighlighted:(BOOL)rippleHighlighted {
     // If ripple becomes highlighted we initiate a ripple with animation.
     [self updateRippleColor];
     [self beginRippleTouchDownAtPoint:_lastTouch animated:_didReceiveTouch completion:nil];
-  } else {
+  } else if (!rippleHighlighted) {
```

Comment:
Is this change necessary for this PR? Can we break it out to a separate PR?

Reply:
This could be a separate PR, but is essential for the initial integration of Ripple with Chips. I can open a separate PR for this one liner. I just thought it wouldn't be a substantial burden to have both improvements (being small) in the same PR.

Reply:
Please break it out to a separate PR; it's not related to the rest of this PR and may risk getting rolled back if the other code above (which is inherently more complex) has to be reverted some day for any reason.

Reply:
Removed